Microsoft Platform Technical (#898)
SkylineCommunications/dataminer-docs-connectors: connector/doc/Microsoft_Platform_Technical.md

If necessary, add more filters, or delete filters using the **Delete** button next to the filtered parameter.

> [!IMPORTANT]
> The Process Counter feature is deprecated and should no longer be used.

> > Please use the new and improved Process Validation feature instead. It provides more accurate and flexible process monitoring capabilities.

Reviewer: I'm not sure if it's clear enough  for a user what you mean here, as it might not be clear what "Process Counter feature" and "Process Validation feature" refer to exactly. Could you rephrase this so it's more clear, especially for a user who is just looking at the UI and doesn't know how things work under the hood?

Reviewer: ```suggestion
> The **Process Counter** subpage is **deprecated** and should no longer be used. Instead, you should use the **Page Validation** subpage, which uses the new and improved Process Validation feature and provides more accurate and flexible process monitoring capabilities.
```
